client/src/AppBundle/Service/Availability/SiriusApiAvailability.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Service\Availability;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class SiriusApiAvailability extends ServiceAvailabilityAbstract
+{
+    public function __construct(ContainerInterface $container)
+    {
+        $this->isHealthy = true;
+
+        try {
+            $response = $container->get('AppBundle\Service\Client\Sirius\SiriusApiGatewayClient')->get('v1/healthcheck');
```

Comment:
You should be able to get this with dependency injection:
```
public function __construct(ContainerInterface $container, SiriusApiGatewayClient $client)
...
$response = $client->get('v1/healthcheck');
```

Reply:
True. I begrudgingly left it as `$container` as it's in keeping with the other Availability services. They are each instantiated manually from within the `ManageController::availabilityAction`, which just passes `$this->container` into each Availability service's constructor. I _could_ inject the `SiriusApiGatewayClient` service into that controller action and pass it in from there, but the same could then be said for each other service. In short, sticking with what we have seemed less contentious.

Reply:
That's fair enough! Makes sense to stay consistent 👍